src/components/VideoCard/styled.js

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+
+const VideoCard = styled.div`
+    display: flex;
+    flex-direction: column;
+    width: 300px;
+    height: 300px;
+    padding: 0 10px;
+`;
+
+const VideoImg = styled.img`
+    width: 200px;
+`;
+
+const VideoTitle = styled.p`
+    background-color: rgba(0, 122, 122, 0.23);
+    z-index: 1;
+    height: 180px;
+`;
+
+const Styled = { VideoCard, VideoImg, VideoTitle };
+export default Styled;
```

Comment:
Same comment as before, you can use the 
```jsx
export const MyComponent = ....
```

this way you can omit the creation of the object and the default export, when you use `export const MyComponent...` JS creates the object for you